lite/tests/classification/conftest.py

```diff
@@ -337,42 +322,40 @@ def classifications_multiclass() -> list[Classification]:
 
 
 @pytest.fixture
-def classifications_multiclass_true_negatives_check() -> list[Classification]:
+def classifications_multiclass_true_negatives_check_animals() -> (
+    list[Classification]
+):
     return [
         Classification(
             uid="uid1",
-            groundtruths=[("k1", "ant")],
-            predictions=[("k1", "ant"), ("k1", "bee"), ("k1", "cat")],
-            scores=[0.15, 0.48, 0.37],
-        ),
-        Classification(
-            uid="uid2",
-            groundtruths=[("k2", "egg")],
-            predictions=[("k2", "egg"), ("k2", "milk"), ("k2", "flour")],
+            groundtruths=["ant"],
+            predictions=["ant", "bee", "cat"],
             scores=[0.15, 0.48, 0.37],
         ),
     ]
 
 
 @pytest.fixture
-def classifications_multiclass_zero_count() -> list[Classification]:
+def classifications_multiclass_true_negatives_check_ingredients() -> (
```

Comment:
We can remove this fixture and its tests b/c it contains the same info as `classifications_multiclass_true_negatives_check_animals`